### DISCUSSION TITLE
[Feature] Automatically updating DB with new folder

### DISCUSSION THREAD

**Discussion by bcsteeve** [immich-app/immich#2723] [Feature Request]
### The feature

In .env file, I defined the library path (eg. /path1).  I renamed the folder to something else (eg. /path2) and updated the .env file.

All the thumbnails remain, but clicking on any image that I hadn't previously clicked on will bring up a blank page.

I _assume_ what's going on is that the change in the path isn't getting into the database for existing images.  Perhaps it should?

Whether or not this is looked at as a potential feature, is there a way for me to fix this without having to start over?

Thanks.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Just to understand exactly what's going on - is this what you did?
1. `UPLOAD_LOCATION=/path1`
2. `mv /path1 /path2`
3. `UPLOAD_LOCATION=/path2`

Did you apply the .env change afterwards? (`docker compose up`)

**Comment by jrasm91** [comment]
You can move files on your hard drive, but they should still be mounted to the same place on the container - always /usr/src/app/upload.
